File: 27-RemoveElement/27-RemoveElement.py
# Last updated: 5/25/2026, 11:21:54 AM
class Solution(object):
    def removeElement(self, nums, val):
        i=0
        for num in nums:
            if num!=val:
                nums[i]=num
                i+=1
        return i
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 5/25/2026, 11:21:46 AM
class Solution {
    public int removeElement(int[] nums, int val) {
        int c=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=val){
                nums[c]=nums[i];
                c++;
            }
        }
        return c;
    }
}